### PR TITLE
Submodule update & Prepare for release 0.10.0.post1

### PR DESCRIPTION
**Pull request by jmoreira-valory**
## DOC SUBMODULE UPDATE PR

**Please, each PO approve this PR after confirming that for the submodule linked to their repos:**

1. The submodule update is pointing to the intended commit, which should be the latest tag/release.

2. The release workflow of the corresponding repository has finished successfully (green icon).

3. To the best of your knowledge, the public-facing tutorial in the submodule (`<submodule_folder>/docs/index.md`) is up to date. In particular, you agree with the instructions in the section "Demo".

4. You have replicated the instructions exactly as described in the public-facing tutorial (with the exact options, hash values, command line parameters, private keys, etc. described there), in a clean virtual environment, and it has worked. Also:
	a. an external user is able to replicate the demo with these instructions, **or**

	b. if this is not 100% possible, (e.g., due to requiring private API keys, registered addresses, etc) this limitation is clearly informed to the user in the tutorial.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+## v0.10.0.post1 (2023-04-03)
+
+Frameworks
+
+* Open Autonomy Framework: open-autonomy https://github.com/valory-xyz/open-autonomy (v0.10.0.post2)
+
+Toolkits
+
+* SMPKit: autonomous-fund - https://github.com/valory-xyz/autonomous-fund (v0.5.0.post1)
+* MintKit: agent-academy-1 - https://github.com/valory-xyz/agent-academy-1 (v0.8.0.post1-3-g116a585)
```
Comment (DavidMinarsch) [inline HISTORY.md:12]: why is the tag so long?

**Review by DavidMinarsch — APPROVED** [review]
Looked at the overall change but did not confirm the accuracy of the subrepos

File: HISTORY.md (hunk 1 above)
Comment (jmoreira-valory) [reply HISTORY.md:12]: It refers to a not-mergeable branch with a ligtweight fix (documentation) after the release took place.

**Review by dvilelaf — APPROVED** [review]

File: HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Release History - Autonolas Documentation
 
+## v0.10.0.post1 (2023-04-03)
```
Comment (0xArdi) [inline HISTORY.md:3]: O believe this should be post2, no?

File: HISTORY.md (hunk 1 above)
Comment (jmoreira-valory) [reply HISTORY.md:12]: This is a different issue.

It is fixed in this PR https://github.com/valory-xyz/agent-academy-1/pull/200/ but hasn't been merged before the release. Thus, there is a DNM branch rooted on the last release that implements this fix in the documentation.

**Pull request merged**